src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public abstract class Task {
+    private final String content;
+    private final boolean done;
+
+    public Task(String content) {
+        this.content = content;
+        this.done = false;
+    }
+
+    public Task(String content, boolean done) {
+        this.content = content;
+        this.done = done;
+    }
```

Comment:
Should `done` be named differently to indicate that it is a boolean variable? I noticed the same issue in several other classes related to `Task`
```suggestion
    public Task(String content, boolean isDone) {
        this.content = content;
        this.isDone = isDone;
    }
```